test/IntegrationTests/SelectiveSamplerTests.cs

```diff
@@ -69,6 +64,8 @@ public void ExportThreadSamples()
         collector.AssertExpectations();
     }
 
+#if NET
+// TODO Implement better tests for .NET Framework, Continuous Profiler is not stopping all threads at once.
```

Comment:
@eftiquar, there is some problem when both continuous profiling and selective sampling is used.
There are no samples exporter by the continuous profiler.

Reply:
Let me check .. thanks for spotting this.

Reply:
I will improve debug level logs for net fx stack capture to note dropped stack samples due to lack of seed or other safety probe failures.